SQLite3/mORMoti18n.pas

```diff
@@ -2225,15 +2250,15 @@ function AddOnceDynArray(const S: WinAnsiString): integer;
 // called within *A() Win32 API -> only english=Ansi text is expected here
 function CB_EnumStringProc(hModule: THandle; lpszType, lpszName: PAnsiChar;
   lParam: PtrInt): Boolean; stdcall;
-var buf: array[0..4095] of AnsiChar;
-    s: WinAnsiString;
+var buf: array[0..4095] of WideChar;
+    s: RawUTF8;
     i: PtrInt;
 begin
   result := true;
   if (PtrInt(lpszType)<>PtrInt(RT_STRING)) then exit;
   i := (PtrInt(lpszName)-1)shl 4;
   for i := i to i+15 do begin // resourcestrings are stored by groups of 16
-    SetString(s,buf,LoadStringA(hInstance,i,buf,sizeof(buf)));
+    SetString(s,buf,LoadStringW(hInstance,i,buf,sizeof(buf)));

```

Comment:
this is IMHO wrong: s is UTF-8 while LoadStringW returns UTF-16 content

Reply:
SetString method causes a problem unless buf is declared as array od WideChar. Please look at callstack
1. LoadStringA and buf is array of AnsiChar
![cb_enumstringproc_1](https://user-images.githubusercontent.com/44496770/47724283-a8144900-dc56-11e8-8439-c5f44c855afa.png)
1. LoadStringW and buf is array od WideChar
![cb_enumstringproc_2](https://user-images.githubusercontent.com/44496770/47724300-b06c8400-dc56-11e8-8c49-7134ce4885cb.png)

Reply:
if s is a RawUTF8 you should NOT use UTF-16 input
use `RawUnicodeToUtf8()` instead of `SetString()`